### DISCUSSION TITLE
Room assistant cluster

### DISCUSSION THREAD

**Discussion by titof2375** [mKeRix/room-assistant#1177] [Q&A]
hello, here I installed room assistant on a VM of ha and zero rpi with wifi.  my problem I don't understand how to do for the cluster, should I put this 
cluster:
   autoDiscovery: true
   port: 6425
   peerAddresses:
     - 192.168.10.10:6425
     - 192.168.10.11:6425 
     on each instance, or only on the leader?

**Comment by jdiegmueller** [comment]
If you have working mDNS between all nodes, you can just run cluster: -- autoDiscovery defaults to true.  If you want to directly specify the peer nodes, you can add the port and peerAddresses as you show above.  You'd want that on both instances.

If you want to influence which node is the cluster leader, you can add weight: <#> where higher <#> gets priority.
